.github/workflows/transfer-issues.yml

```diff
@@ -0,0 +1,66 @@
+name: Transfer issues
+
+# To run this action.
+# 1. Create a personal access token (Classic) with the `repo` and `read:org` scopes.defaults:
```

Comment:
Also need to [authorize it for use with SAML](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on).

Reply:
Added.